src/Resources/contao/library/Contao/DcaExtractor.php

```diff
@@ -401,6 +401,12 @@ protected function createExtract()
 			return;
 		}
 
+		// Return if the DC type is "Folder" and is not database assisted
+		if ($GLOBALS['TL_DCA'][$this->strTable]['config']['dataContainer'] == 'Folder' && empty($GLOBALS['TL_DCA'][$this->strTable]['config']['databaseAssisted']))
```

Comment:
Should I replace it with `(!isset($GLOBALS['TL_DCA'][$this->strTable]['config']['databaseAssisted']) || !$GLOBALS['TL_DCA'][$this->strTable]['config']['databaseAssisted'])`?